packages/react-components/src/components/VideoGallery.test.tsx

```diff
@@ -412,6 +415,92 @@ describe('VideoGallery pinned participants tests', () => {
   );
 });
 
+/* @conditional-compile-remove(vertical-gallery) */
+describe('VideoGallery with vertical overflow gallery tests', () => {
+  beforeAll(() => {
+    mockVideoGalleryInternalHelpers();
+  });
+
+  test('should render remote screenshare and render dominant speaking remote participants in vertical gallery', () => {
```

Comment:
Should we also have tests for 1 participant with video on and all others in VerticalGallery?

Reply:
Added.